Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -195,6 +195,8 @@ PropertyID property_id_from_string(StringView string)
     });
 
     generator.append(R"~~~(
+    if (string.starts_with("--"sv) && string != "--" && string.trim_whitespace() == string)
```

Comment:
Surely we have to do a little more validation here? Not everything that starts with `--*` is a valid custom property :)

Reply:
It's my first time working with a spec, so I didn't fully read the part defining a valid custom property.